src/chttpd/test/eunit/chttpd_bulk_get_test.erl

```diff
@@ -0,0 +1,785 @@
+                                ?ATT := #{<<"stub">> := true}
+                            }
+                        }
+                    }
+                ],
+                <<"id">> := ?DOC
+            }
+        ],
+        Res
+    ).
+
+t_specific_rev({_, DbUrl}) ->
+    Doc = #{<<"id">> => ?DOC, <<"rev">> => <<"2-revb">>},
+    {Code, Res} = bulk_get(DbUrl, [Doc]),
+    ?assertEqual(200, Code),
+    ?assertMatch(
+        [
+            #{
+                <<"docs">> := [
+                    #{
+                        <<"ok">> := #{
+                            <<"_id">> := ?DOC,
+                            <<"_rev">> := <<"2-revb">>,
+                            <<"_attachments">> := #{
+                                ?ATT := #{<<"stub">> := true}
+                            }
+                        }
+                    }
+                ],
+                <<"id">> := ?DOC
+            }
+        ],
+        Res
+    ).
+
+t_specific_rev_latest({_, DbUrl}) ->
+    Doc = #{<<"id">> => ?DOC, <<"rev">> => <<"2-revb">>},
+    {Code, Res} = bulk_get(DbUrl, [Doc], "?latest=true"),
+    ?assertEqual(200, Code),
+    ?assertMatch(
+        [
+            #{
+                <<"docs">> := [
+                    #{
+                        <<"ok">> := #{
+                            <<"_id">> := ?DOC,
+                            <<"_rev">> := <<"2-revb">>,
+                            <<"_attachments">> := #{
+                                ?ATT := #{<<"stub">> := true}
+                            }
+                        }
+                    }
+                ],
+                <<"id">> := ?DOC
+            }
+        ],
+        Res
+    ).
+
+t_ancestor_rev_latest({_, DbUrl}) ->
+    Doc = #{<<"id">> => ?DOC, <<"rev">> => <<"1-reva">>},
+    {Code, Res} = bulk_get(DbUrl, [Doc], "?latest=true"),
+    ?assertEqual(200, Code),
+    ?assertMatch(
+        [
+            #{
+                <<"docs">> := [
+                    #{
+                        <<"ok">> := #{
+                            <<"_id">> := ?DOC,
+                            <<"_rev">> := <<"2-revb">>,
+                            <<"_attachments">> := #{
+                                ?ATT := #{<<"stub">> := true}
+                            }
+                        }
+                    },
+                    #{
+                        <<"ok">> := #{
+                            <<"_id">> := ?DOC,
+                            <<"_rev">> := <<"2-revc">>,
+                            <<"_attachments">> := #{
+                                ?ATT := #{<<"stub">> := true}
+                            }
+                        }
+                    }
+                ],
+                <<"id">> := ?DOC
+            }
+        ],
+        Res
+    ).
+
+t_revs_true({_, DbUrl}) ->
+    Doc = #{
+        <<"id">> => ?DOC,
+        <<"rev">> => <<"1-reva">>
+    },
+    {Code, Res} = bulk_get(DbUrl, [Doc], "?revs=true"),
```

Comment:
Just like with `latest=true` we cheat here and accept most of the same options as a single document GET. We cheated down to the level of accepting parameters in the POST requests (we share the params parsing code), instead of having them as json fields in the request body itself.